feathr-example/installment-payments_credit_card_balance.ipynb

```diff
@@ -0,0 +1,1073 @@
+    "            \"f_SK_ID_PREV\",\n",
+    "            \"f_SK_ID_CURR\",\n",
+    "            \"f_NUM_INSTALMENT_VERSION\",\n",
+    "            \"f_NUM_INSTALMENT_NUMBER\",\n",
+    "            \"f_DAYS_INSTALMENT\",\n",
+    "            \"f_DAYS_ENTRY_PAYMENT\",\n",
+    "            \"f_AMT_INSTALMENT\",\n",
+    "            \"f_AMT_PAYMENT\",\n",
+    "\n",
+    "            \"f_AMT_UNPAID\"\n",
+    "        ], key=key_SK_ID_CURR),\n",
+    "    FeatureQuery(\n",
+    "        feature_list=[\n",
+    "            \"f_SK_ID_CURR_CC\",\n",
+    "            \"f_MONTHS_BALANCE\",\n",
+    "            \"f_AMT_BALANCE\",\n",
+    "            \"f_AMT_CREDIT_LIMIT_ACTUAL\",\n",
+    "            \"f_AMT_DRAWINGS_ATM_CURRENT\",\n",
+    "            \"f_AMT_DRAWINGS_CURRENT\",\n",
+    "            \"f_AMT_DRAWINGS_OTHER_CURRENT\",\n",
+    "            \"f_AMT_DRAWINGS_POS_CURRENT\",\n",
+    "            \"f_AMT_INST_MIN_REGULARITY\",\n",
+    "            \"f_AMT_PAYMENT_CURRENT\",\n",
+    "            \"f_AMT_PAYMENT_TOTAL_CURRENT\",\n",
+    "            \"f_AMT_RECEIVABLE_PRINCIPAL\",\n",
+    "            \"f_AMT_RECIVABLE\",\n",
+    "            \"f_AMT_TOTAL_RECEIVABLE\",\n",
+    "            \"f_CNT_DRAWINGS_ATM_CURRENT\",\n",
+    "            \"f_CNT_DRAWINGS_CURRENT\",\n",
+    "            \"f_CNT_DRAWINGS_OTHER_CURRENT\",\n",
+    "            \"f_CNT_DRAWINGS_POS_CURRENT\",\n",
+    "            \"f_CNT_INSTALMENT_MATURE_CUM\",\n",
+    "            \"f_NAME_CONTRACT_STATUS\",\n",
+    "            \"f_SK_DPD\",\n",
+    "            \"f_SK_DPD_DEF\",\n",
+    "\n",
+    "            \"f_CREDIT_CARD_BALANCE_EMA_AVG\"\n",
+    "        ], key=key_SK_ID_CURR),\n",
+    "]\n",
+    "\n",
+    "\n",
+    "settings = ObservationSettings(\n",
+    "    observation_path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/installments_payments.csv\",\n",
+    "    event_timestamp_column=\"1609472084\",\n",
+    "    timestamp_format=\"epoch\"\n",
+    ")\n",
+    "\n",
+    "client.get_offline_features(observation_settings=settings,\n",
+    "                            feature_query=feature_queries,\n",
+    "                            output_path=\"abfss://feathrhomecreditcafs@feathrhomecreditcasto.dfs.core.windows.net/home_credit_data/output_installment-payment_credit_card_balance.avro\")\n",
+    "client.wait_job_to_finish(timeout_sec=7200)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download the result and show the result\n",
+    "\n",
+    "Let's use the helper function `get_result_df` to download the result and view it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-07-07 10:02:28.837 | INFO     | feathr._synapse_submission:wait_for_completion:134 - Current Spark job status: success\n",
```

Comment:
Logs

Reply:
Clean-up console output of the notebooks.